core/src/main/java/org/apache/brooklyn/core/mgmt/internal/LocalSubscriptionManager.java

```diff
@@ -111,7 +115,7 @@ public long getTotalEventsDelivered() {
             s.subscriberExecutionManagerTagSupplied = true;
         } else {
             s.subscriberExecutionManagerTag = 
-                s.subscriber instanceof Entity ? "subscription-delivery-entity-"+((Entity)s.subscriber).getId()+"["+s.subscriber+"]" : 
+                s.subscriber instanceof Entity ? "subscription-delivery-entity-"+((Entity)s.subscriber).getId() : 
```

Comment:
yes, the tostring repeats the ID, adds the java type which might be nice, but really a bit clunky imo